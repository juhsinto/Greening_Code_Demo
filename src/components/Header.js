import React from "react";

export const Header = ({message}) => {

    console.log("adsfasdfads",message)
    if (message === undefined) {
        message = "A simple Questionnaire"
    }
    

    return (
        <h1 className="header">{message}</h1>
    )
}

export default Header;
