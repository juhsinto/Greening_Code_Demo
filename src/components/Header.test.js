import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header';


it('renders Header correctly with default message', () => {
  const tree = renderer.create(<Header></Header>).toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders Header correctly with custom message', () => {
    let message = "A test message"
    const tree = renderer.create(<Header message={message}></Header>).toJSON();
    expect(tree).toMatchSnapshot();
  });