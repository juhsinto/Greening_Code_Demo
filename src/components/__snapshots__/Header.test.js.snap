// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Header correctly with custom message 1`] = `
<h1
  className="header"
>
  A test message
</h1>
`;

exports[`renders Header correctly with default message 1`] = `
<h1
  className="header"
>
  A simple Questionnaire
</h1>
`;
